test/functional/legacy/074_global_var_in_viminfo_spec.lua

```diff
@@ -0,0 +1,59 @@
+-- Tests for storing global variables in the .viminfo file
+
+local helpers, lfs = require('test.functional.helpers'), require('lfs')
+local clear, execute, eq, neq, eval, wait, spawn =
+  helpers.clear, helpers.execute, helpers.eq, helpers.neq, helpers.eval,
+  helpers.wait, helpers.spawn
+
+describe('storing global variables in viminfo files', function()
+  setup(function()
+    clear()
+    os.remove("Xviminfo")
+  end)
```

Comment:
Although it doesn't matter in this case since there's only 1 `it()`, it's a bit safer to use before_each and after_each in case a test is added later.